### DISCUSSION TITLE
Cannot able to add table feature in wysiwyg text editor

### DISCUSSION THREAD

**Discussion by devimako** [area17/twill#2549] [Q&A]
Need to add table icon in wysiwyg text editor
This is my code i have added the table in this   But table icon is not reflecting in view 
         @formField('wysiwyg', [
            'name' => 'projectdetail_description',
            'label' => 'Apartment Description',
            'placeholder' => 'Text',
            'toolbarOptions' => [
                'bold',
                'italic',
                'underline',
                'strike',
                'blockquote',
                'code-block',
                'code',
                'link',
                'clean',
                'table',
                ['header' => [2, 3, 4, 5, 6, false]],
                'image',
                ['list' => 'bullet'],
                ['list' => 'ordered'],
                ['script' => 'super'],
                ['script' => 'sub'],
                ['align' => []],
            ],
            'translated' => true,
            'note' => 'This field is required *',
            'mediaLibrary' => [
                'enabled' => true,
            ],
        ])

 Twill version: 2.0
 Laravel version: 9.47.0
 PHP version:8.0
 Database engine:postgres

**Comment by ifox** [comment]
Hi @devimako ! You need to use the TipTap editor using `'type' => 'tiptap'` on your wysiwyg field to use the table feature.
